crates/ruff_python_parser/tests/snapshots/invalid_syntax@re_lex_logical_token.py.snap

```diff
@@ -832,7 +832,16 @@ Module(
    |
 28 | # The lexer is nested with multiple levels of parentheses
 29 | if call(foo, [a, b
-   |                   ^ Syntax Error: Expected `]`, found NonLogicalNewline
+30 |     def bar():
+   |     ^^^ Syntax Error: Expected `]`, found `def`
```

Comment:
I find those error messages more accurate. We only expect a `]` because `def` is the start of a new keyword. It's otherwise perfectly fine to write a list over multiple lines.

Reply:
Hmm, but then the following error message points at this location "expecting `)`, found `Newline`". I think this is still an improvement but just found this a bit inconsistent. No need to change anything here.